DTS-STN/Service-Canada-Labs
# Storybook 7 migration

# Description

This task is to migrate from Storybook 6.4 to Storybook 7. You can read about Storybook 7 [here](https://storybook.js.org/blog/7-0-beta/) and I recommend you do it's cool stuff!

This should fix the problems with storybook builds failing and help us update related dependencies.

## Acceptance Criteria

yarn build-storybook should successfully build storybook in static-storybook folder.

yarn storybook should successfully run an instance of storybook

## Test Instructions

1. Run `yarn build-storybook`
2. Build should occur successfully without errors
3. Run `yarn storybook`
4. Instance of storybook should work without breaking errors

## Checklist
